src/compiler/gcc.rs

```diff
@@ -73,25 +73,25 @@ impl CCompilerImpl for GCC {
 }
 
 ArgData!{ pub
-    TooHardFlag(()),
+    TooHardFlag,
     TooHard(OsString),
     PassThrough(OsString),
     PassThroughPath(PathBuf),
-    PreprocessorArgumentFlag(()),
+    PreprocessorArgumentFlag,
```

Comment:
That's much nicer! I fiddled with this for a bit but couldn't find a working solution except with the unstable `?` macro repetition operator.